lib/generators/rolemodel/optics/base/templates/app/views/layouts/application.html.slim

```diff
@@ -0,0 +1,13 @@
+doctype html
```

Comment:
This file should be in the slim generator not in Optics

Reply:
right, I forgot about that.  Thanks

Reply:
That being said, it might be helpful to modify the application layout to work according to what Optics gives you. Technically not a requirement to use that though.